pmd-java/src/main/java/net/sourceforge/pmd/lang/java/rule/codestyle/UnnecessaryLocalBeforeReturnRule.java

```diff
@@ -134,7 +136,7 @@ private boolean isInitDataModifiedAfterInit(final VariableNameDeclaration variab
                             // Variable found! Check usage locations
                             for (final NameOccurrence occ : entry.getValue()) {
```

Comment:
I know this is really old… but I just realized, we are checking the symbol table, and it hadn't picked up the usage in member selection. Even though we fixed the rule, we still seem to have a bug in the symbol table for PMD 6 / a new test case for the PMD 7 rework